docs/source/reference-lowlevel.rst

```diff
@@ -49,7 +49,11 @@ attributes, :meth:`trio.Lock.statistics`, etc.). Here are some more.
 Global statistics
 -----------------
 
-.. autofunction:: current_statistics
+.. function:: current_statistics() -> RunStatistics
+
+   Returns an object containing run-loop-level debugging information:
+
+.. autoclass:: trio.RunStatistics()
```

Comment:
I manually added the docs here to make it so that the list of attributes wasn't repeated - since the class right below is going to show those. But for the docstring, it makes sense to repeat them since you're probably only looking at the function.